src/executor/JobExecutor.ts

```diff
@@ -22,18 +22,16 @@ export class JobExecutor {
     this.stopped = true;
   }
 
-  async execute(jobEntity: JobEntity, parameters?: JobParameters, force: boolean = false): Promise<JobResult> {
```

Comment:
Can you give a bit more detail as to why we're removing the `force` option here? I might be reading the code wrong, but it looks like now we _can't_ run a job if it's `maxRunning` has been reached.

Reply:
Yes, which would be the expected behavior, right?
The force flag was added with the "only one schedule shall be active" PR (#489) and I do not remember why :see_no_evil:. Since it was not mentioned anywhere, there is no issue for it, and we couldn't come up with a use case we removed it again.

Reply:
Hm, yeah I guess. I believe we added it back in the day in order to allow people to just run their jobs regardless of any constraints they may have added earlier. But if there's no use-case for it, removing it sounds fine.